src/components/Editor/HighlightLine.js

```diff
@@ -52,6 +52,15 @@ export class HighlightLine extends PureComponent<Props> {
       prevProps.selectedSource
     );
 
+    if (
+      !selectedLocation.forced &&
+      prevProps.selectedLocation.line &&
+      selectedLocation.line &&
+      prevProps.selectedLocation.line === selectedLocation.line
+    ) {
+      return;
+    }
+    selectedLocation.forced = false;
```

Comment:
Not sure if this should be done through an action or something.

Reply:
lol, this seems a bit forced.

I'd prefer to use the `prevLine` hack than to pass a highlight flag into selectLocation